src/common/com/intellij/plugins/haxe/ide/annotator/HaxeSemanticAnnotator.java

```diff
@@ -44,26 +47,35 @@ public void annotate(PsiElement element, AnnotationHolder holder) {
   static void analyzeSingle(final PsiElement element, AnnotationHolder holder) {
     if (element instanceof HaxePackageStatement) {
       PackageChecker.check((HaxePackageStatement)element, holder);
-    } else if (element instanceof HaxeMethod) {
+    }
+    else if (element instanceof HaxeMethod) {
       MethodChecker.check((HaxeMethod)element, holder);
-    } else if (element instanceof HaxeClass) {
+    }
+    else if (element instanceof HaxeClass) {
       ClassChecker.check((HaxeClass)element, holder);
-    } if (element instanceof HaxeType) {
+    }
+    if (element instanceof HaxeType) {
```

Comment:
The clauses above all have 'else if' while this line and the one below (line #60) only have an 'if' statement.  Either construct should work because the types are all independent.  The else clause is probably better, though.  At least we should make them consistent.